packages/expo-payments-stripe/android/build.gradle

```
@@ -61,7 +61,7 @@ dependencies {
 
   api 'com.google.android.gms:play-services-wallet:15.0.1'
   api 'com.google.firebase:firebase-core:16.0.1'
-  api 'com.stripe:stripe-android:8.1.0'
+  api 'com.stripe:stripe-android:16.1.1'
```

Actually we can try using `safeExtGet` and global variable so that people can override this version.
```suggestion
  api "com.stripe:stripe-android:${safeExtGet('stripeVersion', '16.1.1')}"
```

---

good idea!